src/ScriptCs.Core/App_Packages/LibLog.3.1/LibLog.cs

```diff
@@ -0,0 +1,1941 @@
+//===============================================================================
```

Comment:
1900 lines of code in one file really? Is there no refactored version? We can have a logging folder and put individual files there.